components/Modal/index.js

```diff
@@ -17,16 +16,16 @@ const Modal = ({ text, buttonText, next, counter, nav }) => {
         overlayStyle={styles.modal}
         isVisible={visible}
         onBackdropPress={() => {
+          onPress();
           toggleOverlay();
-          next();
         }}>
         <Text style={styles.header}>{text}</Text>
         <Button
           buttonStyle={styles.button}
           title={buttonText}
           onPress={() => {
-            counter > 9 ? nav.navigate(HOME_SCREEN) : toggleOverlay();
```

Comment:
Not using this Modal right now, but I changed it to accept just a onPress function for reusability down the line